src/mars.d

```diff
@@ -596,7 +595,10 @@ extern (C++) int tryMain(size_t argc, const(char)** argv)
             else if (strcmp(p + 1, "g") == 0)
                 global.params.symdebug = 1;
             else if (strcmp(p + 1, "gc") == 0)
-                global.params.symdebug = 2;
+            {
+                deprecation(Loc(), "use -g instead of -gc");
+                global.params.symdebug = 1;
```

Comment:
While you're deprecating, it should still set it to 2. The deprecation message is enough.